src/utils/account.js

```diff
@@ -0,0 +1,16 @@
+import { getCurrentSanbaseSubscription } from './plans'
+import { PRO } from '../components/Navbar/NavbarProfileDropdown'
+
+export const checkIsProUser = data => {
+  const sanbaseSubscription = getCurrentSanbaseSubscription(data)
+
+  return sanbaseSubscription && sanbaseSubscription.plan
+    ? sanbaseSubscription.plan.name === PRO
+    : false
+}
+
+export const checkIsProState = ({ user: { data } }) => {
```

Comment:
It can be just "isUserPro" or "isPro" @GarageInc

Reply:
State is 100% must be removed

Reply:
Agree. But it is only simplified code. I imported copy paste from project in one place with 2 functions, because in some places we are checking state, in other places we are checking only user.